src/lib/backend.ai-client-node.ts

```diff
@@ -617,6 +617,21 @@ class Client {
     return this._wrapWithPromise(rqst);
   }
 
+  /**
+   *  Update user's fullname.
+   *  
+   *  CAUTION: username attribute is unique key. 
+   *           therefore full_name should be changed, not the username.
+   */
+  async update_username(oldFullname, newFullname) {
+    let body = {
+      'old_full_name': oldFullname,
+      'new_full_name': newFullname
+    };
+    let rqst = this.newSignedRequest('POST', `/auth/update-fullname`, body);
+    return this._wrapWithPromise(rqst);
+  }
+
```

Comment:
I think it would be better to use already-existing graphQL mutation to update user's information unless there is a good reason.